modules/developers-guide/pages/concepts/governance.adoc

```diff
@@ -4,28 +4,29 @@
 :IC: Internet Computer
 :company-id: DFINITY
 
-As a decentralized platform, all changes to the configuration and behavior of the Internet Computer are controlled by a governance body called the Network Nervous System (NNS). The NNS controls many aspects of the Internet Computer platform including the following:
+As a decentralized blockchain, all changes to the configuration and behavior of the Internet Computer are controlled by a governance body called the Network Nervous System (NNS). The NNS controls many aspects of the Internet Computer blockchain including the following:
 
 - which data center providers participate in the network
 - the number, location, and ownership of the nodes accepted from a data center provider
-- assignment of nodes to subnets
-- whether upgrades to smart contract canister software or management services are allowed or not
+- assignment of nodes to subnet blockchains
+- whether upgrades to canister smart contracts or management services are allowed or not
```

Comment:
What are management services? Maybe replace  ..or  to a new protocol version ...

Reply:
I prefer that